src/Proto.Persistence.SqlServer/SqlServerProvider.cs

```diff
@@ -234,6 +234,8 @@ private void ExecuteNonQuery(string sql)
 
             using var command = new SqlCommand(sql, connection);
 
+            connection.Open();
```

Comment:
This was the main problem for me. When I first initialized the persistence provider I got an exception that the connection was not open.  The rest of the places in this class that we use the SQL Connection object we open it explicitly but, for some reason, this one didn't have a call to open the connection.  This change seems to correct the issue.